src/infra/ChatAnalyticsReporter.ts

```diff
@@ -14,12 +15,15 @@ export class ChatAnalyticsReporter {
   private readonly apiKey: string;
   /** The endpoint to report events to, inferred by the env and region */
   private readonly endpoint: string;
+  /** Whether to enable session tracking for analytics events */
+  private readonly sessionTrackingEnabled: boolean;
 
   constructor(
-    { apiKey, env, region }: ChatAnalyticsConfig,
+    { apiKey, env, region, sessionTrackingEnabled = false }: ChatAnalyticsConfig,
```

Comment:
@yen-tt  Will the default behavior be `true` when invoked inside of other libraries like chat-ui-react?

Reply:
was going to ask you about that once this is done! I can have default behavior in Chat SDK set to true if that's what we want. My general assumption is that this should be something user opt-in instead of other way around, so I have analytics repo set it to false by default.

Reply:
I think that we should probably make it opt out for Chat specifically just because, if you don't set it, you lose out completely on metrics like Conversation Rate. We did the same thing for the theme IIRC. Session tracking was default on, but could be toggled off.

Reply:
I see, then I can default to true in this repo. This config is for Chat specifically anyway

Reply:
updated! PR now have sessionTrackingEnabled default to true

Reply:
Great!